instrumentation/apache-log4j-1/src/main/java/com/nr/agent/instrumentation/log4j1/Logger_Instrumentation.java

```diff
@@ -0,0 +1,17 @@
+package com.nr.agent.instrumentation.log4j1;
+
+import com.newrelic.api.agent.NewRelic;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+import org.apache.log4j.Priority;
+
+@Weave(originalName = "org.apache.log4j.Category")
+public class Logger_Instrumentation {
```

Comment:
Class name should match the name of the class being weaved, so we'll want to change this to `Category_Instrumentation`